admin_app/src/app/page.tsx

```diff
@@ -105,7 +105,7 @@ export default function Home() {
 
   // functions to edit and add content
   const editCard = (card: Content) => {
-    setCardToEdit(card);
+    setCardToEdit({ ...card });
```

Comment:
By splatting the object, we get it's properties. By putting it in curly brackets, we create a new object with those properties.

Reply:
Neat!! Copilot:
> The line setCardToEdit({ ...card }); is using the spread operator (...). This operator is used to create a new object with the same properties as the card object ... but is a separate object. This means that if you modify card later, cardToEdit will not be affected because they are different objects.